grammar/ballerina.configuration.json

```diff
@@ -11,7 +11,8 @@
     ["{", "}"],
     ["[", "]"],
     ["(", ")"],
-    ["\"", "\""]
+    ["\"", "\""],
+    ["<", ">"]
   ],
   "surroundingPairs": [
     ["{", "}"],
```

Comment:
shall we also add [`] since this is required for templating?

Reply:
+1

Reply:
Added with https://github.com/ballerina-platform/plugin-vscode/pull/202/commits/eb38c6ae63d48977efcb4011fe27e1e60079cb8a